lib/plug/parsers.ex

```diff
@@ -200,23 +200,26 @@ defmodule Plug.Parsers do
       nil ->
         %{conn | body_params: %{}}
     end
+    merge_params(conn)
```

Comment:
So maybe instead of calling merge_params here, we should only call it after we set body_params? This way we will keep the previous behaviour where body_params may not always be set?